--- .venv/lib/python3.11/site-packages/sphinx/domains/python/__init__.py.old	2024-10-23 10:51:56.882646206 +0100
+++ .venv/lib/python3.11/site-packages/sphinx/domains/python/__init__.py	2024-10-23 10:53:16.557626613 +0100
@@ -204,6 +204,7 @@

     option_spec: ClassVar[OptionSpec] = PyObject.option_spec.copy()
     option_spec.update({
+        'signal': directives.flag,
         'abstractmethod': directives.flag,
         'async': directives.flag,
         'classmethod': directives.flag,
@@ -219,6 +220,9 @@
         if 'final' in self.options:
             prefix.append(nodes.Text('final'))
             prefix.append(addnodes.desc_sig_space())
+        if 'signal' in self.options:
+            prefix.append(nodes.Text('signal'))
+            prefix.append(addnodes.desc_sig_space())
         if 'abstractmethod' in self.options:
             prefix.append(nodes.Text('abstract'))
             prefix.append(addnodes.desc_sig_space())
