--- .venv/lib/python3.X/site-packages/sphinx/domains/python.py	2022-03-08 11:00:06.152347159 +0000
+++ .venv/lib/python3.X/site-packages/sphinx/domains/python.py	2022-03-08 11:03:07.292940495 +0000
@@ -768,6 +768,7 @@

     option_spec: OptionSpec = PyObject.option_spec.copy()
     option_spec.update({
+        'signal': directives.flag,
         'abstractmethod': directives.flag,
         'async': directives.flag,
         'classmethod': directives.flag,
@@ -787,6 +788,9 @@
         if 'final' in self.options:
             prefix.append(nodes.Text('final'))
             prefix.append(addnodes.desc_sig_space())
+        if 'signal' in self.options:
+            prefix.append(nodes.Text('signal'))
+            prefix.append(addnodes.desc_sig_space())
         if 'abstractmethod' in self.options:
             prefix.append(nodes.Text('abstract'))
             prefix.append(addnodes.desc_sig_space())
